core/templates/pages/library-page/search-bar/search-bar.component.html

```diff
@@ -40,16 +38,18 @@
         role="menu">
       <ng-container *ngFor="let item of selectionDetails.categories.masterList">
         <li ngbDropdownItem *ngIf="selectionDetails.categories.selections[item.id]">
-          <a (click)="toggleSelection('categories', item.id)" *ngIf="!first && !last" class="dropdown-item">
+          <a (click)="toggleSelection('categories', item.id)" (keydown.enter)="toggleSelection('categories', item.id)" tabindex="0" *ngIf="!first && !last" class="dropdown-item">
             <span class="e2e-test-selected">{{ item.text | translate }}</span>
             <i *ngIf="selectionDetails.categories.selections[item.id]" class="fas fa-check float-right oppia-search-bar-category-selection-symbol"></i>
           </a>
-          <a (click)="toggleSelection('categories', item.id)" *ngIf="first" (keydown)="onMenuKeypress($event, 'category', {shiftTab: ACTION_CLOSE})" class="dropdown-item">
+          <a (click)="toggleSelection('categories', item.id)" (keydown.enter)="toggleSelection('categories', item.id)" tabindex="0" *ngIf="first" (keydown)="onMenuKeypress($event, 'category', {shiftTab: ACTION_CLOSE})" class="dropdown-item">
```

Comment:
Ditto

Reply:
Not done? (break this line onto multiple lines.

Reply:
Done